project-2-e-commerce/src/helpers/index.js

```diff
@@ -61,3 +61,38 @@ export const getAddCartMessage = (item) => {
 export const getSum = (a, b) => {
     return Number(a) + Number(b)
 }
+
+export const validatePaymentForm = (obj) => {
+    let errMessage = ''
+    let fieldErrName = ''
+    const email_regex = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,3})+$/
+    const vnf_regex = /((09|03|07|08|05)+([0-9]{8})\b)/g;
```

Comment:
Regex cho vào file config hoặc constant
Những biến có giá trị không đổi này thì để như sau
```suggestion
    export const EMAIL_REGEX = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,3})+$/
    export const VNF_REGEX = /((09|03|07|08|05)+([0-9]{8})\b)/g;
```